### DISCUSSION TITLE
OpenTelemetry support for Firefly?

### DISCUSSION THREAD

**Discussion by phandox** [firefly-iii/firefly-iii#9248] [General]
Hi, 

I wonder if someone has experience integrating OpenTelemetry to Firefly-III? I thought of using zero-code instrumentation - https://opentelemetry.io/docs/zero-code/php/ but I wonder if someone tried already and found some pitfalls?

My use case is to gather spans and have them in Grafana for finding slow calls when interacting with Firefly III.

**Comment by JC5** [comment]
I've never done it before, did you try it already?

**Reply by phandox** [reply]
Not yet - when I'll have more time on hand I can try it out. However I am not a PHP developer and I consider tracing to be more focused on app developers, as they are more aware where the manual instrumentation in code belong.
